src/main/java/com/sonyericsson/jenkins/plugins/bfa/db/KnowledgeBase.java

```diff
@@ -51,6 +51,8 @@
  */
 public abstract class KnowledgeBase implements Describable<KnowledgeBase>, Serializable {
 
+    private static final long DEFAULT_FAILURE_CAUSES = 0;
```

Comment:
It should contain the notion that it is the **number of failure** causes to ease confusion.
For example name it similar to the method where it's used, e.g. `DEFAULT_NBR_OF_NULL_FAILURE_CAUSES`